how-tos/saml.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: "SAML"
+sidebarTitle: "SAML / Enterprise SSO"
+---
+
+PropelAuth's [SAML SSO](https://www.propelauth.com/post/what-does-saml-actually-mean) support allows your users to
+connect their organizations their own identity provider (e.g. Okta, Azure AD, JumpCloud, etc).
```

Comment:
Should the YT video be on here?

Reply:
I had switched the two SAML docs we have, moved them around and have the correct doc here now, which includes the video